Source/Global/mem.h

```diff
@@ -48,20 +48,35 @@ class http_memory_buffer
 
 NAMESPACE_XBOX_HTTP_CLIENT_END
 
+template<typename TAllocator>
+struct http_alloc_deleter
```

Comment:
this doesn't need to be templated on the allocator type, just the object type. It would save some typing elsewhere

Reply:
I would also add a template alias for the allocator traits to avoid having to repeat that everywhere

Reply:
done